prow/plugins/updateconfig/updateconfig.go

```diff
@@ -90,7 +90,7 @@ func (g *OSFileGetter) GetFile(filename string) ([]byte, error) {
 }
 
 // Update updates the configmap with the data from the identified files
```

Comment:
Consider updating the doc to describe what bootstrap does

Reply:
Done.